### PR TITLE
qodana fix : code smell, missing returns.

### PR DESCRIPTION
**Pull request by oussamanhairech**
qodana fix : code smell, missing returns.

### CODE REVIEW

**Comment by oussamanhairech** [comment]
qodana fix : code smell, missing returns.

File: htdocs/comm/action/class/cactioncomm.class.php [hunk 1]
```diff
@@ -341,5 +341,6 @@ public function getNomUrl($withpicto = 0)
 		if ($transcode != "Action".$this->code) {
 			return $transcode;
 		}
+		return -1;
```
Comment (eldy) [inline htdocs/comm/action/class/cactioncomm.class.php:344]: So comment doc say "@return string Label of action type"
So you must choose between making a return '';
or a return -1 and you modify the doc to say  "* @return int|string Label of action type"

File: htdocs/conf/conf.phy~ [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?php
```
Comment (eldy) [inline htdocs/conf/conf.phy~:1]: You must remove this file from your commit

File: htdocs/conf/conf.phz~ [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?php
```
Comment (eldy) [inline htdocs/conf/conf.phz~:1]: You must remove this file from the PR

File: htdocs/core/class/fiscalyear.class.php [hunk 4]
```diff
@@ -446,6 +446,8 @@ public function LibStatut($status, $mode = 0)
 				return $langs->trans($this->statuts_short[$status]).' '.img_picto($langs->trans($this->statuts_short[$status]), 'statut6');
 			}
 		}
+		return "";
+
```
Comment (eldy) [inline htdocs/core/class/fiscalyear.class.php:450]: No spaces before a }

File: htdocs/core/class/google.class.php [hunk 5]
```diff
@@ -86,5 +86,7 @@ public function getGeoCoordinatesOfAddress($address)
 		//echo "Longitude: $longitude ',' Latitude: $latitude";
 
 		$i++;
+		return "Longitude: $longitude, Latitude: $latitude";
+
```
Comment (eldy) [inline htdocs/core/class/google.class.php:90]: No spaces before a }

File: htdocs/comm/mailing/class/advtargetemailing.class.php [hunk 6]
```diff
@@ -516,6 +516,7 @@ public function savequery($user, $arrayquery)
 				$this->create($user);
 			}
 		}
+		return -1;
```
Comment (eldy) [inline htdocs/comm/mailing/class/advtargetemailing.class.php:519]: When we reach this point, it seems this is the "standard" path. So we should not return error but ok, so
return 1

Comment (eldy) [reply htdocs/comm/mailing/class/advtargetemailing.class.php:519]: @oussamanhairech ping

**Pull request closed without merging**